ex1-ex2/index.js

```diff
@@ -0,0 +1,64 @@
+EX1;
+
+class Cars {
+  constructor(brand, model, color, mileage) {
+    this.brand = brand;
+    this.model = model;
+    this.color = color;
+    this.mileage = mileage;
+  }
+
+  getCarDetails() {
+    return "Cars[brand = ${this.brand}, model = ${this.model}, color = ${this.color}, mileage = ${this.mileage}]";
+  }
+}
+
+function displayCars() {
+  const car1 = new Cars("VOLVO", "XC-60", "Black", "234.000");
```

Comment:
```suggestion
  const car1 = new Cars("VOLVO", "XC-60", "Black", 234_000);
```

Ultimul parametru se asteapta a fi un numar, nu un string.
caracterul "_" este optional si are efect doar vizual (pentru ca numarul sa se citeasca mai bine)